api/items_post.md

```diff
@@ -115,16 +115,41 @@ The example JSON payload below describes all the required and optional params th
               "post": []
             },
 
-            // Optional: args
+            // (DEPRECATED) Optional: args
             // List of values of positional arguments to the method/function call
             // NOTE: as this can contain sensitive data, you may want to scrub the values
             "args": ["<Request object>", 25],
 
-            // Optional: kwargs
+            // (DEPRECATED) Optional: kwargs
             // Object of keyword arguments (name => value) to the method/function call
             // NOTE: as this can contain sensitive data, you may want to scrub the values
             "kwargs": {
               "force": true
+            },
+
+            // Optional: argspec
+            // List of the names of the arguments to the method/function call.
+            "argspec": [request, user],
+
+            // Optional: varargspec
+            // Name of the list containing the unnamed arguments to the method/function call.
+            // THe actual list will be found in locals.
+            "varargspec": args,
```

Comment:
Ah, but those are the most common names in Python.

`*args` -> args
`**kwargs` -> kwargs